client-src/elements/chromedash-toast.ts

```diff
@@ -74,35 +71,39 @@ class ChromedashToast extends LitElement {
   /**
    * Shows a message in the toast.
    * @method showMessage
```

Comment:
I think we can also delete the `@method`, which is redundant with the method name.
```suggestion
```